applications/concept-catalogue/src/test/kotlin/ConceptRegistrationApplicationTest.kt

```diff
@@ -122,4 +121,281 @@ class ConceptRegistrationApplicationTests {
+        begrep.kildebeskrivelse.forholdTilKilde = Kildebeskrivelse.ForholdTilKildeEnum.EGENDEFINERT
+        uriTekst.tekst = "tekst"
+        begrep.kildebeskrivelse.addKildeItem(uriTekst)
+        assertTrue(validator(begrep))
+
+        begrep.kildebeskrivelse = Kildebeskrivelse()
+        begrep.kildebeskrivelse.forholdTilKilde = Kildebeskrivelse.ForholdTilKildeEnum.EGENDEFINERT
+        uriTekst.uri = ""
+        uriTekst.tekst= ""
+        begrep.kildebeskrivelse.addKildeItem(uriTekst)
+        assertFalse(validator(begrep))
+
+        begrep.kildebeskrivelse = Kildebeskrivelse()
+        begrep.kildebeskrivelse.forholdTilKilde = Kildebeskrivelse.ForholdTilKildeEnum.EGENDEFINERT
+        uriTekst.uri = "uri"
+        uriTekst.tekst= "tekst"
+        begrep.kildebeskrivelse.addKildeItem(uriTekst)
+        assertTrue(validator(begrep))
+    }
+
+    @Test
+    fun testBegrepMerknadValidation() {
+        val validator = { b: Begrep -> BegrepValidator.validate(b) }
+        val begrep = createBegrep()
+        begrep.status = Status.GODKJENT
+
+        begrep.merknad = null
+        assertTrue(validator(begrep))
+
+        begrep.merknad = ""
+        assertFalse(validator(begrep))
+
+        begrep.merknad = "merknad"
+        assertTrue(validator(begrep))
+    }
+
+    @Test
+    fun testBegrepAnsvarligVirksomhetValidation() {
+        val validator = { b: Begrep -> BegrepValidator.validate(b) }
+        val begrep = createBegrep()
+        begrep.status = Status.GODKJENT
+
+        begrep.ansvarligVirksomhet = null
+        assertFalse(validator(begrep))
+
+        begrep.ansvarligVirksomhet = createTestVirksomhet().apply {
+            id = null
+        }
+        assertFalse(validator(begrep))
+
+        begrep.ansvarligVirksomhet = createTestVirksomhet().apply {
+            id = ""
+        }
+        assertFalse(validator(begrep))
+
+        begrep.ansvarligVirksomhet = createTestVirksomhet().apply {
+            id = "123"
+        }
+        assertFalse(validator(begrep))
+
+        begrep.ansvarligVirksomhet = createTestVirksomhet().apply {
+            id = "123456789"
+        }
+        assertFalse(validator(begrep))
+
+        begrep.ansvarligVirksomhet = createTestVirksomhet()
+        assertTrue(validator(begrep))
+
+        begrep.ansvarligVirksomhet = createTestVirksomhet().apply {
+            id = "943574537"
+        }
+        assertTrue(validator(begrep))
+    }
+
+    @Test
+    fun testBegrepEksempelValidation() {
+        val validator = { b: Begrep -> BegrepValidator.validate(b) }
+        val begrep = createBegrep()
+        begrep.status = Status.GODKJENT
+
+        begrep.eksempel = null
+        assertTrue(validator(begrep))
+
+        begrep.eksempel = ""
+        assertFalse(validator(begrep))
+
+        begrep.eksempel = "eksempel"
+        assertTrue(validator(begrep))
+    }
+
+    @Test
+    fun testBegrepFagområdeValidation() {
+        val validator = { b: Begrep -> BegrepValidator.validate(b) }
```

Comment:
(bare spørsmål for avklaring).
Her er regelen at fagområde KAN være null, men om det ikke er det så skal det være en streng ?